metacat-common-server/src/main/java/com/netflix/metacat/common/server/usermetadata/LookupService.java

```diff
@@ -33,31 +34,39 @@
      * @param name lookup name
      * @return lookup
      */
-    Lookup get(String name);
+    default Lookup get(String name) {
+        return null;
+    }
 
     /**
      * Returns the value of the lookup name.
      *
      * @param name lookup name
      * @return scalar lookup value
      */
-    String getValue(String name);
+    default String getValue(String name) {
```

Comment:
You should probably flag the return of these as `@Nullable` so calling methods are prepared. That or we should throw "Unsupported operation"